src/util/tests/evalUnits.test.ts

```diff
@@ -91,4 +85,10 @@ describe('Tests suite - evalUnits', () => {
     expect(actualValue).toBeFalsy;
     expect(msg).toBe(EvalMessages.GIBBERISH);
   });
+
+  it('Should accept as input a HUGE number and return error message', () => {
```

Comment:
I'd rather see this handled correctly, but yeah, better than crashing :)

Reply:
You will now see it handled correctly